subcommands/waves/init.go

```diff
@@ -28,6 +29,14 @@ Use a "fioctl wave complete <wave>" to update all devices (make it globally avai
 Use a "fioctl wave cancel <wave> to cancel a wave (make it no longer available).`,
 		Run:  doInitWave,
 		Args: cobra.ExactArgs(3),
+		Example: `
+To simply initialize a wave run the following:
+$ fioctl wave init -k ~/path/to/keys/targets.only.key.tgz wave-name 4 production
```

Comment:
I'd paraphrase to a more instructive language here: `Start a new wave for the target version 4 and the 'production' device tag:`

Reply:
good suggestion. I think katrina has been trying to get us to stop using works like "simply".

Reply:
I adjusted it. :+1: